internal/provider/resources/passwordconfig.go

```diff
@@ -160,6 +162,37 @@ func (r *passwordConfigResource) Schema(_ context.Context, _ resource.SchemaRequ
 	}
 }
 
+func (r passwordConfigResource) ValidateConfig(ctx context.Context, req resource.ValidateConfigRequest, resp *resource.ValidateConfigResponse) {
+	var data passwordConfigModel
+	diags := req.Config.Get(ctx, &data)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// If the projectID isn't yet known, skip validation for now.
```

Comment:
(non-blocking) why exit early? the rest isn't dependent on knowing it.

Reply:
Good point -- initially I was doing a project lookup here for other values, but it's no longer used, so this check is irrelevant.